### PR TITLE
[FIX] calendar: notification detail correct redirection

### PR DESCRIPTION
**Pull request by fdamhaut**
Step to reproduce:
- Create an event in calendar with reminder
- (Move the event to trigger the reminder)
- Click on 'Details'

Current Behaviour:
- Open a new event

Behaviour after PR:
- Open the event linked to the notification

opw-2761492

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by fdamhaut** [comment]
@arj-odoo

File: addons/calendar/static/src/js/services/calendar_notification_service.js [hunk 1]
```diff
@@ -61,11 +61,12 @@ export const calendarNotificationService = {
...
-                                        "calendar.action_calendar_event_notify",
-                                        {
-                                            resId: notif.event_id,
-                                        }
+                                    await action.doAction({
+                                        type: 'ir.actions.act_window',
+                                        res_model: 'calendar.event',
+                                        res_id: notif.event_id,
+                                        views: [[false, 'form']],
+                                    }
```
Comment (arj-odoo) [inline addons/calendar/static/src/js/services/calendar_notification_service.js:69]: Why can't you update the parameters to keep using action_calendar_event_notify but fix the call?
action_calendar_event_notify has no domain or context, active_id could be passed an the domain could use that. 
action_calendar_event_notify is not super helpful so we could drop it in master and use the same strategy as you.

Comment (fdamhaut) [reply addons/calendar/static/src/js/services/calendar_notification_service.js:69]: No matter what I tried, I couldn't open the event with `calendar.action_calendar_event_notify` and as you said it's only used once elsewhere to create a new event so no more info there + I didn't want to modify too much in Stable but I could had a todo/ remove it in master during the FW port if you believe it's best.

Comment (arj-odoo) [reply addons/calendar/static/src/js/services/calendar_notification_service.js:69]: Yes, that would be nice. We don't need it anymore.

**Comment by fdamhaut** [comment]
@arj-odoo :D

**Comment by arj-odoo** [comment]
@robodoo r+

**Pull request closed without merging**